test/gui/tst_syncing/test.feature

```diff
@@ -7,7 +7,7 @@ Feature: Syncing files
     Background:
         Given user "Alice" has been created on the server with default attributes and without skeleton files
 
-    @smokeTest
+    @smokeTest @skip
```

Comment:
Could you also mention issue number to track
```suggestion
    @smokeTest @skip @issue-0000
```

Reply:
done

Reply:
done